notes/3.1.0.markdown

```diff
@@ -0,0 +1,29 @@
+ScalikeJDBC 3.1.0 is out for Scala 2.12, 2.11 and 2.10.
+
+http://scalikejdbc.org/
+
+![ScalikeJDBC Logo](http://scalikejdbc.org/images/logo.png)
+
+### Changes
+
+- [core] #705 add to{DateTimeClass}WithZoneId methods in UnixTimeInMillisConverter by @xuwei-k
+- [core] #709 Replaced type check with pattern matching @by tototoshi
```

Comment:
I am sorry for a late comment. Is it a typo of `by @totoshi`?

Reply:
Ah, you're right. I will fix it.